content/internal/delivery/content/grpc.go

```diff
@@ -0,0 +1,73 @@
+package content
+
+import (
+	"context"
+
+	"github.com/dranikpg/dto-mapper"
+	"github.com/go-park-mail-ru/2023_1_ContentDealers/content/pkg/proto/content"
+)
+
+type Grpc struct {
+	content.UnimplementedContentServiceServer
+
+	useCase UseCase
+}
+
+func NewGrpc(useCase UseCase) *Grpc {
+	return &Grpc{useCase: useCase}
+}
+
+func (service *Grpc) GetFilmByContentID(ctx context.Context, contentID *content.ContentID) (*content.Film, error) {
```

Comment:
https://gist.github.com/adamveld12/c0d9f0d5f0e1fba1e551#variable-names

For a method receiver, one or two letters is sufficient.